pkg/vulnloader/nvdloader/convert.go

```diff
@@ -152,14 +152,18 @@ func toBaseMetricV2(metrics []*apischema.CVEAPIJSON20CVSSV2) *jsonschema.NVDCVEF
 }
 
 func toBaseMetricV3(metrics30 []*apischema.CVEAPIJSON20CVSSV30, metrics31 []*apischema.CVEAPIJSON20CVSSV31) *jsonschema.NVDCVEFeedJSON10DefImpactBaseMetricV3 {
-	switch {
-	case len(metrics31) != 0:
-		return toBaseMetricV31(metrics31)
-	case len(metrics30) != 0:
-		return toBaseMetricV30(metrics30)
-	default:
-		return nil
+	// Prefer CVSS 3.1.
+	baseMetric := toBaseMetricV31(metrics31)
+	if baseMetric != nil {
+		return baseMetric
+	}
+
+	baseMetric = toBaseMetricV30(metrics30)
+	if baseMetric != nil {
+		return baseMetric
 	}
+
+	return nil
```

Comment:
Didn't like the len checks?

Reply:
didn't seem very necessary, and would have caused for more if/else, which is nice to avoid for readability :)